### ISSUE TITLE
[Feature] Add Quit Stash option from the Web UI

### ISSUE THREAD

**Issue by skier233** [stashapp/stash#3078]
**Is your feature request related to a problem? Please describe.**
I wasn't aware it was possible to quit stash other than through task manager until I asked and someone pointed out that it's possible with the windows task bar in the bottom right. I hardly ever use this toolbar because so few apps use it and the ones that do always allow you to perform the same functionality elsewhere. I think it makes sense to add this same functionality within the webapp as well for consistency so users like me that weren't aware of it's existence still have a way to close stash.

**Describe the solution you'd like**
Adding an option to the UI (maybe right next to the log out button in the top right) to quit stash

**Describe alternatives you've considered**
Currently just killing with task manager. This could cause to data corruption though.

**Additional context**

**Comment by DogmaDragon** [comment]
Browser sandboxing prevents this by design for security reasons. Just imagine clicking a random link on some website and it's killing your browser or some other process running on your operating system. 

It would probably require installing additional service with OS-level access and required permissions to achieve that.

**Comment by fornivea** [comment]
> Browser sandboxing prevents this by design for security reasons. Just imagine clicking a random link on some website and it's killing your browser or some other process running on your operating system.
> 
> It would probably require installing additional service with OS-level access and required permissions to achieve that.

Not really, you could make a /shutdown endpoint or GraphQL mutation that shuts down the application.

**Comment by QxxxGit** [comment]
Yeah I close via terminal which might be in the middle of a process/task if I'm not careful. I suppose you'd have to consider whether or not to cancel tasks immediately or wait until the processes are finished before quitting (I support the latter). And with the future development of users, it would require owner permissions. Not sure what else would need to be considered.

And yes, a /shutdown endpoint that shuts down safely (finish tasks, close connection, etc) seems ideal.

**Comment by fornivea** [comment]
When you use CTRL+C in a terminal you're sending SIGINT which should be handled in the application as a graceful shutdown.

Regardless I think the ask is a valid one, applications such as qBittorrent also have a way to exit the app from the web UI & it should be easy to implement
